functions/lib/emails/cancel_event.py

```
@@ -0,0 +1,72 @@
+        "transactionalId": LOOPS_PAYMENT_CANCELLED_EMAIL_TEMPLATE_ID,
+        "email": email,
+        "dataVariables": {
+            "name": full_name,
+            "eventName": event_name,
+            "orderId": order_id,
+            "ticketCount": str(ticket_count),
+        },
+    }
+
+    try:
+        response = requests.post(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)